src/tools/core/ensemble_stat/ensemble_stat.cc

```diff
@@ -194,6 +198,11 @@ int met_main(int argc, char *argv[]) {
    // Perform verification
    process_vx();
 
+  // Save the CTRACK metrics 
+  #ifdef WITH_PROFILER
+  ctrack::result_print();
```

Comment:
@bikegeek if `ctrack::result_print()` is already being called every time in `basic/vx_util/main.cc` don't we NOT need to call it in the application code? Won't this cause the results to be printed twice?

Reply:
If ctrack::result_print() is NOT in the application code, the benchmarking information for the application code does not get saved.